src/App/Containers/Users/TableUsers.js

```diff
@@ -12,18 +13,20 @@ const UsersTableModule = ({ userDatas }) => {
     </tr>
   ))
   return (
-    <Table dark striped hover>
-      <thead>
-        <tr>
-          <th>User ID</th>
-          <th>Name</th>
-          <th>Email ID</th>
-          <th>City</th>
-        </tr>
-      </thead>
+    <div style={{ width: '90%', padding: '10px', margin: '20px' }}>
+      <Table dark striped hover>
```

Comment:
You can create a general <Table /> component with columns and data as props, so that you can reuse the general table component through out the application. We are passing different column names as props to the table.